resources/assets/js/features/car/actionTypes.js

```diff
@@ -1,5 +1,8 @@
-export const VEHICLE_GET_ALL_SUCCESS = 'VEHICLE_GET_ALL_SUCCESS';
-export const VEHICLE_GET_SUCCESS = 'VEHICLE_GET_SUCCESS';
-export const VEHICLE_CREATE_SUCCESS = 'VEHICLE_CREATE_SUCCESS';
-export const VEHICLE_EDIT_SUCCESS = 'VEHICLE_EDIT_SUCCESS';
-export const VEHICLE_DELETE_SUCCESS = 'VEHICLE_DELETE_SUCCESS';
\ No newline at end of file
+export const GET_BRANDS_DATA = 'GET_BRANDS_DATA';
```

Comment:
add CAR_ or VEHICLE_ prefix to all actions

Reply:
@TatsuUkraine fixed it